### PR TITLE
Adds higher order functions for elm

### PR DESCRIPTION
**Pull request by gera-cameron**
Addresses: #1  for elm.

### CODE REVIEW

**Comment by mtso** [comment]
Thanks for your pull request! Can you please link me to a working online Elm playground with your code?

I'm getting ```I cannot find a `Html` type:``` at https://ellie-app.com/3B8rzyZvWfba1. Is the associated HTML set up incorrectly?

**Comment by gera-cameron** [comment]
Oops! Sorry about that. I took a look at what you had and updated it. Check out https://ellie-app.com/3BnbfbnwnXYa1 for a compiling example.

File: elm.elm [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module Elm exposing (main)
+
+import Html
```
Comment (mtso) [inline elm.elm:3]: Can you update the imports to explicitly expose the imported identifiers?
```
import Html exposing (Html, text, div)
import String exposing (fromInt)
import List
```

**Comment by mtso** [comment]
Just the imports and it looks good to go :)

**Comment by mtso** [comment]
#18 will be used in lieu of this

**Pull request closed without merging**